javascript/.eslintrc.json

```diff
@@ -125,6 +125,7 @@
     "array-bracket-spacing": ["error", "never"],
     "object-curly-spacing": ["error", "always"],
     "no-extra-semi": "error",
-    "react/no-deprecated": "off"
+    "react/no-deprecated": "off",
+    "react/jsx-uses-vars": [2]
```

Comment:
What does `[2]` mean in this context? 
Any way we can clarify like how the others use `"error"` or `"always"`?